src/main/java/Entities/Address.java

```diff
@@ -0,0 +1,79 @@
+package Entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlRootElement;
+@XmlRootElement(name="address")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class Address {
+
+  @JsonProperty("id")
+  private byte id;
+
+  @JsonProperty("street")
+  private String street;
+
+  @JsonProperty("city")
+  private String city;
+
+  @JsonProperty("state")
+  private String state;
+
+  @JsonProperty("zip_code")
+  private int zipCode;
+
+  public Address(@JsonProperty("id") byte id,@JsonProperty("street") String street, @JsonProperty("city")String city, @JsonProperty("state")String state, @JsonProperty("zip_code")int zipCode) {

```

Comment:
Whenever you use @XmlAccessorType(XmlAccessType.FIELD) on top of the class you don't really need to assign annotations on constructors and methods.